### DISCUSSION TITLE
nerdctl does not show any running containers

### DISCUSSION THREAD

**Discussion by tomikonio** [rancher-sandbox/rancher-desktop#4935] [Q&A]
Hi,
Using Rancher desktop `1.8.1` with nerdctl on Windows 11.
Kubernetes is running fine, but I have the following issue:
`nerdctl ps` does not show anything running, same result when using it in an ubuntu wsl machine, windows powershall and inside the rancher-desktop wsl as well.

Is this supposed to be like that?
If so, how can we view the containers that run in the cluster without `kubectl` commands?

**Comment by jandubois** [comment]
`containerd` has the concept of namespaces (which are separate from kubernetes namespaces). Containers you run with `nerdctl run` are in the `default` namespace, but containers started by kubelet are in `k8s.io`. To see them you have to run:

```
nerdctl -n k8s.io ps
```
